jquery.recent-supporters.js

```diff
@@ -266,9 +266,21 @@
       // if there is at least one supporter more loaded then shown
       if (supporterElements.length > settings.maxSupportersVisible) {
         var minHeight = $container.height();
-        $container.css({minHeight: minHeight + 'px'});
-        $('li:visible', $ul).last().slideUp({duration: 200, easing: settings.cycleEasing});
-        supporterElements.last().hide().detach().prependTo($ul).slideDown({duration: 200, easing: settings.cycleEasing});
+        $container.css({
+          minHeight: minHeight + 'px',
+          maxHeight: minHeight + 'px'
```

Comment:
Should we rename the variable as it is now used for both min&max height?

Reply:
I think it’s ok to keep it, it still sets the min-height permanently (the max-height just for 200 milliseconds)

Reply:
So it’s more like the `initalHeight` or `heightBeforeAnimation`?

Reply:
Well, it takes the current height and sets it as minimum height so the list can gain height but not shrink again. Should we use `currentHeight` then?

Reply:
Anything that’s helps the reader more than `minHeight` (which doesn’t help at all) is an improvement.

Reply:
I’ve updated the commit and added comments too!